corehq/form_processor/submission_post.py

```diff
@@ -166,12 +168,25 @@ def _get_success_message(self, instance, cases=None):
             return '   √   '
 
         user = CouchUser.get_by_user_id(instance.user_id)
-        messages = [_("'{form_name}' successfully saved!")
```

Comment:
does not what we show on submit response come from FP today  in `submitResponseMessage` ?